### PR TITLE
Revert "1203-db-migration-delete-answer-with-9998"

### PR DESCRIPTION
**Pull request by billli0**
Reverts beyondessential/tupaia#1502 because it needs to be tested.

### CODE REVIEW

**Review by Alastair-L — APPROVED** [review]
Approving, including merging to dev without testing after Kostas gives approval.

**Comment by kael89** [comment]
No need to reverse this if we can test it directly in dev. I suspect this could be possible - I'll check with the testers

**Pull request closed without merging**
